src/main/java/com/slimebot/commands/config/engine/ConfigCategoryCommand.java

```diff
@@ -6,15 +6,21 @@
 import java.lang.reflect.Field;
 
 public class ConfigCategoryCommand extends BaseCommand<CommandContext> {
-	public ConfigCategoryCommand(ConfigCategory category, Field[] fields, InstanceProvider instanceProvider) {
-		description = category.description();
+    public ConfigCategoryCommand(ConfigCategory category, Field[] fields, InstanceProvider instanceProvider) {
+        description = category.description();
 
-		for (Field field : fields) {
-			ConfigField info = field.getAnnotation(ConfigField.class);
+        for (Field field : fields) {
+            ConfigField info = field.getAnnotation(ConfigField.class);
 
-			if (info == null) continue;
+            if (info == null) continue;
 
-			addSubcommand(info.command(), new ConfigPropertyCommand(field, info, category, instanceProvider));
-		}
-	}
+            if (info.type() == ConfigFieldType.CHANNEL_LIST || info.type() == ConfigFieldType.ROLE_LIST || info.type() == ConfigFieldType.NUMBER_LIST || info.type() == ConfigFieldType.STRING_LIST) {
```

Comment:
Either info.type().toString().contains("LIST") or a new 'isList' variable in the ConfigFieldType enum